variables.tf

```diff
@@ -14,10 +14,69 @@
  * limitations under the License.
  */
 
-variable "project_id" {
-  description = "The project ID to deploy to"
+variable "name" {
+    type = string
+    description = "The resource name for the Dataset.s"
 }
 
-variable "bucket_name" {
-  description = "The name of the bucket to create"
+variable "project" {
+    type = string
+    description = "The ID of the project in which the resource belongs. If it is not provided, the provider project is used."
```

Comment:
We should mandate explicitly setting a project, since that's what we usually do on CFT.

Reply:
Done.